llms/llama/convert.py

```diff
@@ -140,6 +139,21 @@ def quantize(weights, config, args):
     return quantized_weights, quantized_config
 
 
+def make_shards(weights: dict, max_file_size_GiB: int = 15):
```

Comment:
style nit: `max_file_size_gb`

Reply:
as we are using 2**30 = GiB I'd suggest to use: `max_file_size_gibibyte` as I find `max_file_size_gb` wrong and `max_file_size_gib` unreadable.